pkg/provisioner/ironic/ironic.go

```diff
@@ -420,65 +420,13 @@ func (p *ironicProvisioner) ValidateManagementAccess(credentialsChanged bool) (r
 			imageData = p.host.Spec.Image
 		}
 
-		checksum, checksumType, ok := imageData.GetChecksum()
-
+		_, _, ok := imageData.GetChecksum()
```

Comment:
I could image moving this into `getImageUpdateOptsForNode()` (where we're already calling it to get the _other_ two return values) at some point, and only checking here whether `updates` is empty.

Reply:
Yeah I actually did that initially but it breaks a bunch of tests - I could perhaps make that change in a follow-up PR to avoid combining refactoring with test updates and logical changes in the same PR?

Reply:
I had a closer look and found a way to avoid breaking the tests, so this is addressed in the latest version